resources/testdata/data/json/timezones/names-long@1/und.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can you verify if it is correct that these files are empty?  (If so, that's great)

Reply:
Yes, some of the file should be empty. Here is the complete list from our test data:

```
generic-long@1/und.json
specific-long@1/und.json

generic-short@1/th.json
generic-short@1/fr.json
generic-short@1/und.json
generic-short@1/ru.json
generic-short@1/ja.json
generic-short@1/tr.json

specific-short@1/ar.json
specific-short@1/ccp.json
specific-short@1/fr.json
specific-short@1/th.json
specific-short@1/ar-EG.json
specific-short@1/und.json
specific-short@1/ru.json
specific-short@1/bn.json
specific-short@1/tr.json
```